contrib/scripts/build-images.sh

```diff
@@ -0,0 +1,251 @@
+#!/bin/bash
```

Comment:
Could we move this into the sriov operator repo?

Reply:
The script to build and push the images is not specific to SRIOV. It probably could also work in a similar way for e.g. dpu-operator. It's also unclear, how useful in general -- while it's useful in the context of CDA.

Reply:
If it's meant for cda, we should use python.

Reply:
I will split the patch, and drop the script.